### PR TITLE
Download python.exe

### PR DESCRIPTION
**Pull request by benjaoming**

### CODE REVIEW

File: windows/make.bat [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+:CANNOT_FIND
+ECHO Cannot find the wheel package...
+
+:DOWNLOAD_PYTHON
+SET VERSION=3.4.3
+SET PYTHON_URL=https://www.python.org/ftp/python/%VERSION%/python-%VERSION%
+SET PYTHON_DIR=%~dp0python-setup\python-%VERSION%
+
+BITSADMIN /TRANSFER python-%VERSION%.msi /DOWNLOAD /PRIORITY NORMAL "%PYTHON_URL%.msi" "%PYTHON_DIR%.msi"
+BITSADMIN /TRANSFER python-%VERSION%.amd64.msi /DOWNLOAD /PRIORITY NORMAL "%PYTHON_URL%.amd64.msi" "%PYTHON_DIR%.amd64.msi"
```
Comment (benjaoming) [inline windows/make.bat]: @mrpau-julius should this still be in active?

File: windows/make.vbs [hunk 2]
```diff
@@ -1,2 +0,0 @@
-Set oShell = WScript.CreateObject("WSCript.shell")
-oShell.run "cmd /K title Building the installation package... |inno-compiler\ISCC.exe installer-source\KolibriSetupScript.iss & exit"
```
Comment (benjaoming) [inline windows/make.vbs]: Since this file is now empty, should it be deleted? @mrpau-julius

File: windows/python-setup/python-exe.bat [hunk 3]
```diff
@@ -1,13 +1,18 @@
 @echo off
+
+Set version=3.4.3
```
Comment (benjaoming) [inline windows/python-setup/python-exe.bat]: Not necessary to fix now, but next time we do a version bump of the bundled Python installer, let's instead change the filename to just `python3-installer-32bit.msi` so we don't have to worry about this hard-coded version string.

File: windows/make.bat (hunk 1 above)
Comment (mrpau-julius) [reply windows/make.bat]: @benjaoming yes it should be active. This script is for Windows Developer user.

File: windows/make.vbs (hunk 2 above)
Comment (mrpau-julius) [reply windows/make.vbs]: @benjaoming yes it must be deleted.

File: windows/python-setup/python-exe.bat (hunk 3 above)
Comment (mrpau-julius) [reply windows/python-setup/python-exe.bat]: Ok, @benjaoming, I will find ways, next time on how to version bump of the bundled Python installer.

**Comment by mrpau-richard** [comment]
@benjaoming I tested the installer built with this branch and it's working as expected. I think we can merge this and create another tag for the Kolibri 0.12 new Windows installer release. Let's also update the value of `KOLIBRI_WINDOWS_INSTALLER_VERSION` variable with the new tag using this PR [5330](https://github.com/learningequality/kolibri/pull/5330)

**Comment by benjaoming** [comment]
I can't review this PR because I opened it :) But I'm all good with it, great work @mrpau-julius :)

**Comment by mrpau-richard** [comment]
Thanks for that @benjaoming . Let's merge this PR and create another tag for this PR [5330](https://github.com/learningequality/kolibri/pull/5330).

**Pull request merged**